src/EdFi.AnalyticsMiddleTier.DataStandard32/Asmt/PostgreSQL/0016-View-StudentAssessmentFact-Alter.sql

```diff
@@ -0,0 +1,132 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
```

Comment:
I think you forgot to replicate some of the changes you made in the SQL Server version to this Postgres version.